acmw-db-app/components/AttendanceForm.jsx

```diff
@@ -41,9 +22,24 @@ const AttendanceForm = (props) => {
                 <h1>Attendance Form</h1>
                 <p>Thank you for attending an ACM-W event! Please fill out this form to record your attendance!</p>
                 <h2>Event Information</h2>
-                <TextField label="Event Code" onChange={(event) => setEventCode(event.target.value)}/>
-                <TextField label="First Name" onChange={(event) => setFirstName(event.target.value)}/>
-                <TextField label="Last Name.#" onChange={(event) => setLastName(event.target.value)}/>
+                <TextInput
```

Comment:
I'd prefer if the extra blank lines were deleted

Reply:
wait actually I don't even know if they're there, maybe it's a GitHub formatting issue. But either way, approved!